src/SWP/Component/MultiTenancy/Model/AbstractOrganization.php

```diff
@@ -0,0 +1,90 @@
+<?php
```

Comment:
There is no need for an AbstractOrganization class, I think. Just an Organization class. If it's not possible to have an Organization without a tenant.